client/post-editor/post-editor.jsx

```diff
@@ -782,6 +801,14 @@ const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
+		// Update global state to reflect newly saved post and reset all
+		// previous edits (preserving type).
+		this.props.receivePost( post );
+		this.props.resetPostEdits( post.site_ID );
```

Comment:
Since we are grabbing the `siteId` in the connect statement, should we just pass it in as a prop and utilize here?